File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/Results.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Results {

    @SerializedName("next")
    @Expose
    private String next;
    @SerializedName("items")
    @Expose
    private List<Item> items = null;

    public String getNext() {
        return next;
    }

    public void setNext(String next) {
        this.next = next;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverExplore/Results.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore

import com.google.gson.annotations.Expose

class Results {
    @SerializedName("next")
    @Expose
    private var next: String? = null

    @SerializedName("items")
    @Expose
    private var items: List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Item>? = null
    fun getNext(): String? {
        return next
    }

    fun setNext(next: String?) {
        this.next = next
    }

    fun getItems(): List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Item>? {
        return items
    }

    fun setItems(items: List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Item>?) {
        this.items = items
    }
}